### DISCUSSION TITLE
Add Share button functionality (with plugin?) for social media

### DISCUSSION THREAD

**Discussion by holgerflick** [squidfunk/mkdocs-material#5654] [Ideas]
It would be great if the blog module could offer a share button of some kind to share blog posts on Twitter, LinkedIn, Facebook etc.

Especially for the site creator it would make it easier to publish notifications on different social media.

**Comment by squidfunk** [comment]
You can create a new [change request](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/), if you like, and we'll see if other users might find that helpful as well 😊

**Comment by mweth** [comment]
I'm interested in this also.  Will find existing request or create new.

**Comment by mweth** [comment]
@holgerflick I created a change request here.  https://github.com/squidfunk/mkdocs-material/issues/6736

Please comment and upvote if it matches your thoughts.
